libraries/pbrlib/genglsl/lib/mx39_microfacet.glsl

```diff
@@ -0,0 +1,30 @@
+#include "libraries/pbrlib/genglsl/lib/mx_microfacet.glsl"
```

Comment:
We always bring in the 1.38 code because we systematically re-use all the code that was not changed in 1.39.

Modus operandi:
For any code file that was modified in 1.39:
- Copy it verbatim, but with a mx39 prefix

Then, for all definitions in those code files that had a change in 1.39:
- Prefix the identifier with `mx39_` (functions), `MX39_` (macros), `Mx39` (structs), identify all uses in Mx39 files and replace at call site with the prefixed version.

Long and tedious, but it works in the end.